README.md

```diff
@@ -1 +1,198 @@
+
+- **[Usage of flexbox]**
+- **[Mobile responsives setup]**
+- **[Use-of-HTML5 tools]**
+- **[Follow HTML and CSS best practice]**
+- **[Dynamically designed]**
+- **[DOM Manipulation]**
+- **[Scroldown menu for mobile version]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [<a href="https://titelunda.github.io/Capstone1/" target="_blank">Online version</a>](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-Visual Studio Code
+-Github Account
+-Git Bash
+-Node JS
+-Behance Template
+
+### Setup
+
+Clone this repository to your desired folder:
+Use git clone command or downloaded ZIP folder
+Example command:
+
+```sh
+  cd my-folder
+  git clone: (https://github.com/TiteLunda/Capstone1.git)
+```
+
+### Install
+
+Install this project with: npm Example commands:
+
+```sh
+  cd my-project
+  npm init -y
+```
+
+```sh
+  npm install --save-dev hint@7.x
+```
+
+```sh
+  npx hint .
+```
+
+```sh
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
+
+### Usage
+
+To run the project, execute the following command: npm test
+Example command:
+
+```sh
+   GitHub Pages Server
+```
+
+### Run tests
+
+To run tests, run the following command: npm test
+Example command:
+
+```sh
+  npx stylelint "**/*.{css,scss}" --fix
+```
+```sh
+  npx eslint "**/*.js" --fix
+```
+
+### Deployment
+
+You can deploy this project using:
+GitHub Pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Umair Hamza**
+
+- GitHub: [@TiteLunda](https://github.com/TiteLunda)
+- Twitter: [@LundaTite](https://twitter.com/LundaTite)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tite-lunda-094956199/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[More section inclusiveness]**
+- [ ] **[Desktop Responsivness]**
+- [ ] **[Animation]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [https://github.com/TiteLunda/Portofolio_Tite/issues/5](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project then please follow me on Linkedin, Twitter and GitHub. Greetings!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for this project and all my partner for their contribution!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Great job @TiteLunda 🤝

- [ ] Kindly ensure your README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).